concepts/Common/Licensing/15 License Key Expiration and How to Update.md

```diff
@@ -0,0 +1,5 @@
+A license key is supplied with each specific major version. You do not have to change license key for any versions within one major: a minor version, beta, etc. When switching to a new major version, the license key must be updated. 
```

Comment:
**minor**
As for me, the main idea is that these things are coupled, not that they are provided
```suggestion
A license key is bound to a major version. You do not have to change license key for any versions within one major: a minor version, beta, etc. When switching to a new major version, the license key must be updated. 
```